### PR TITLE
Reverted breaking change in PR #1049

### PR DESCRIPTION
**Pull request by andrea-sdl**
## Description

This PR reverts the code on PR #1049 while we investigate it further.

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
1. Run `npm install`
1. Run `npm run build`
1. Run `node dist/bin/vip-app list` runs fine.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@automattic/vip",
-  "version": "2.19.0",
```
Comment (pschoffer) [inline package.json]: It shouldn't revert this I think.

**Review by pschoffer — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (andrea-sdl) [reply package.json]: you're right, updating the code

Comment (andrea-sdl) [reply package.json]: @pschoffer fixed

**Review by pschoffer — APPROVED** [review]
Thank you!

**Pull request merged**